vars/neoDeploy.groovy

```diff
@@ -196,7 +216,7 @@ def call(parameters = [:]) {
 
 private getNeoExecutable(configuration) {
 
-    def neoExecutable = 'neo.sh' // default, if nothing below applies maybe it is the path.
+    def neoExecutable = NEO_DEFAULT_CMD // default, if nothing below applies maybe it is the path.
 
     if (configuration.neoHome) {
         neoExecutable = "${configuration.neoHome}/tools/neo.sh"
```

Comment:
maybe this occurance of neo.sh should also use NEO_DEFAULT_CMD

Reply:
done.